west.yml

```diff
@@ -98,7 +98,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 478f03a71256aeb2890e685ebf5a1dc6832a6b5f
+      revision: pull/48/head
```

Comment:
This needs to be updated to use a SHA before merging.

Reply:
The hal PR was merged, so you can update the SHA now.

Reply:
Done.